analysis_driver/reader/version_reader.py

```diff
@@ -0,0 +1,73 @@
+import subprocess
+import yaml
+
+from analysis_driver.exceptions import AnalysisDriverError
+from analysis_driver.config import default as cfg
+
+
+class Version_reader():
+    tool_name = None
+    command = None
+    def __init__(self, tool_name, command, prefix=None):
```

Comment:
`prefix` is never used to set `self.prefix`, so `_get_stdout_from_command` will crash.